jig.py

```diff
@@ -0,0 +1,72 @@
+"""Jig"""
+from datetime import timedelta
+import random
+from time import time
+from numpy import mean
+import plotly.graph_objects as go
+from model.library.functions import clamp
+from model.library.objects import AMI, DataPoint, SampleData
+from run import model
+
+# TODO: Mean of results over the set of number of runs
+
+# Repeat the simulation with the same samples
+# TODO: array of samples replace number of samples
+NUMBER_OF_SAMPLES = 10
+samples = [SampleData(clamp(random.gauss(0.90, 0.2), 0.01, 0.99),
```

Comment:
Run this file, this is the main entry for starting the model, this line is where you can configure the samples that are passed into the sample, the NUMBER_OF_SAMPLES variable also effects the mean print at the end so the only thing you will need to change is this as well as the run line